sql/mob_pools.sql

```diff
@@ -275,7 +275,7 @@ INSERT INTO `mob_pools` VALUES (214,'Arboricole_Spider','Arboricole_Spider',235,
 INSERT INTO `mob_pools` VALUES (215,'Arcane_Phantasm','Arcane_Phantasm',115,0x0500380200000000000000000000000000000000,10,10,2,240,100,0,1,1,0,2,0,0,0,3,0,0,6,1,0,115);
 INSERT INTO `mob_pools` VALUES (216,'Archaic_Chariot','Archaic_Chariot',63,0x00001D0700000000000000000000000000000000,1,1,6,320,100,0,23,0,2,2,0,0,7,129,0,0,0,1,0,63);
 INSERT INTO `mob_pools` VALUES (217,'Archaic_Chest','Archaic_Chest',183,0x0000020100000000000000000000000000000000,1,1,7,170,100,0,1,0,0,0,0,0,0,0,0,0,0,1,0,183);
-INSERT INTO `mob_pools` VALUES (218,'Archaic_Gear','Archaic_Gear',119,0x00001B0700000000000000000000000000000000,1,1,12,240,100,0,23,1,1,2,0,0,1,1153,0,0,0,1,0,119);
+INSERT INTO `mob_pools` VALUES (218,'Archaic_Gear','Archaic_Gear',119,0x00001B0700000000000000000000000000000000,1,1,12,240,100,0,23,1,1,2,0,0,1,1153,0,0,0,1,0,120);
```

Comment:
my fave part is where list ID 119 doesn't even exist. Do singular gears have diff skills from multi gears? if so we might need to make list 119 someday lol